iceoryx_examples/iceperf/mq.cpp

```diff
@@ -34,24 +34,18 @@ MQ::MQ(const std::string& publisherName, const std::string& subscriberName) noex
 void MQ::cleanupOutdatedResources(const std::string& publisherName, const std::string& subscriberName) noexcept
 {
     auto publisherMqName = PREFIX + publisherName;
-    iox::posix::posixCall(mq_unlink)(publisherMqName.c_str())
-        .failureReturnValue(ERROR_CODE)
-        .ignoreErrnos(ENOENT)
-        .evaluate()
-        .or_else([&](auto& r) {
-            std::cout << "mq_unlink error for " << publisherMqName << ", " << r.getHumanReadableErrnum() << std::endl;
-            exit(1);
-        });
+    IOX_POSIX_CALL(mq_unlink)
+    (publisherMqName.c_str()).failureReturnValue(ERROR_CODE).ignoreErrnos(ENOENT).evaluate().or_else([&](auto& r) {
+        std::cout << "mq_unlink error for " << publisherMqName << ", " << r.getHumanReadableErrnum() << std::endl;
+        exit(1);
+    });
```

Comment:
I wouldn't say I like the formatting here, but nothing you can change.

Reply:
It would be great to get something like this
```cpp
IOX_POSIX_CALL(mq_unlink)(publisherMqName.c_str())
    .failureReturnValue(ERROR_CODE)
    .ignoreErrnos(ENOENT).evaluate()
    .or_else([&](auto& r) {
        std::cout << "mq_unlink error for " << publisherMqName << ", " << r.getHumanReadableErrnum() << std::endl;
        exit(1);
    });
```
... maybe chatGPT can create a clang-format configuration for this :laughing: